### PR TITLE
Ad type for Nokta Ad Server is added for amp-ad

### PR DESCRIPTION
**Pull request by merty**
Nokta Ad Server serves 100M ads per day. We are one of the biggest Internet companies in Turkey. This server is used by some high volume sites such izlesene.com, sinemalar.com, blogcu.com, alkislarlasiyorum.com, and some other sites that reach 70% of Turkey's Internet traffic.

We want to integrate our sites with AMP. For a quick and clean integration; we want Nokta Ad Server as a supported network for **amp-ad** component.

### CODE REVIEW

**Comment by merty** [comment]
I signed it!

**Comment by ferrerodbgm** [comment]
@lannka  Yes, it was. The CLA belongs to Noktacom Medya İnternet Hizmetleri San. ve Tic. A.Ş.. dev-cla@nokta.com is our mail for this CLA.

/cc @jridgewell

**Comment by erwinmombay** [comment]
@ferrerodbgm was it just recently signed? i dont see it in the system yet, might take a few hours

**Comment by ferrerodbgm** [comment]
It was signed 7 hours ago. I also didn't see the CLA in my CLAs page. However I try to fill the form and try to save again; the page says this CLA is already signed.

**Comment by ferrerodbgm** [comment]
@erwinmombay It was signed 7 hours ago. I also didn't see the CLA in my CLAs page. However I try to fill the form and try to save again; the page says this CLA is already signed.

**Comment by ferrerodbgm** [comment]
@erwinmombay according to https://cla.developers.google.com/about that page, this confirmation process can take a few days. How can we speed this up?

/cc @lannka

**Comment by merty** [comment]
I signed it!

**Comment by ferrerodbgm** [comment]
@lannka render-start support is added. We are waiting for reviews. Thanks.

File: extensions/amp-ad/amp-ad.md [hunk 1]
```diff
@@ -123,6 +123,7 @@ resources in AMP. It requires a `type` argument that select what ad network is d
 - [Mixpo](../../ads/mixpo.md)
 - [Nativo](../../ads/nativo.md)
 - [Nend](../../ads/nend.md)
+- [Nend](../../ads/nokta.md)
```
Comment (lannka) [inline extensions/amp-ad/amp-ad.md]: oops :-)

File: examples/ads.amp.html [hunk 2]
```diff
@@ -667,6 +668,16 @@ <h2 id="nend">Nend</h2>
     <div placeholder></div>
     <div fallback></div>
   </amp-ad>
+  
+  <h2 id="nokta">Nokta</h2>
+  <amp-ad width="300" height="250" 
+	  type="nokta"
```
Comment (lannka) [inline examples/ads.amp.html]: nitpick: pls keep the indentation the same as other examples

File: ads/nokta.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+import {writeScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function nokta(global, data) {
+  validateData(data, [], ['category', 'site', 'zone']);
```
Comment (lannka) [inline ads/nokta.js]: so i suppose there is no mandatory fields?

Comment (ferrerodbgm) [inline ads/nokta.js]: @lannka there were 3 mandatory fields. i change that file and md file

File: examples/ads.amp.html (hunk 2 above)
Comment (ferrerodbgm) [inline examples/ads.amp.html]: the indentation is changed

File: extensions/amp-ad/amp-ad.md (hunk 1 above)
Comment (ferrerodbgm) [inline extensions/amp-ad/amp-ad.md]: ah, sorry :) i fixed that with last change

**Pull request merged**